exercises/practice/triangle/triangle.f90

```diff
@@ -0,0 +1,12 @@
+
+module triangle
+  implicit none
+
+  contains
+
+  logical function equilateral(tria)
+    integer,dimension(3) :: tria
```

Comment:
We can afford to use full names, it's 2021 and we don't really need to save bytes in source code.

Instead of `tria` what do you think about `triangle` or `points`?

Reply:
@hiljusti is right, but it isn't just about memory, it is about being able to look up words in a dictionary if it is not understood.  If geometry or trigonometry is not my history, then I might think that "tria" is something that "math people" know.  I might try to look it up in my dictionary, but would get little help.  Then I would pick up the phone and ask the author of that line of code, at 3am, because I need to know.  We can all get more sleep if this is a real word.

Reply:
I agree but I took the names from the problem specification which then probably should also be updated.

Reply:
I think [the problem specification](https://github.com/exercism/problem-specifications/tree/main/exercises/triangle) only defines the function names (`equilateral`, which I'm fine with) and not the parameter names (`tria`, which I think can be `triangle`)

Reply:
I cannot use the word "triangle" because that is the module name, this is the reason or tria. I could maybe change it to "triangle_in" or "triangle_edges". I think I will use the latter.